src/random.cr

```diff
@@ -105,13 +119,27 @@ module Random
     end
   end
 
+  # Returns a random `Float64` in the given *range*.
+  #
+  # ```
+  # Random.new.rand(6.2..21.768) # => 15.2989
+  # ```
+  def rand(range : Range(Float, Float)) : Float64
+    span = range.end - range.begin
+    if span > 0 || !range.excludes_end?
+      range.begin + (range.excludes_end? ? rand(span) : rand * span)
+    else
+      raise ArgumentError.new "incorrect rand value: #{range}"
+    end
+  end
+
   # see `#rand`
   def self.rand : Float64
     DEFAULT.rand
   end
 
   # see `#rand(x)`
-  def self.rand(x) : Int32
+  def self.rand(x)
```

Comment:
Not sure about that. If you pass x as `Int32`, the result is `Int32`. If you pass x as `Float64` it will be `Float64`. The return type depends on x. Of course the return type will be `Int32 ` or `Float64`, but showing that the return type is `Int32 | Float64` will be confusing for user, as most probably they will have to deal with just a single return type. I would remove that.